css/style.css

```diff
@@ -1,18 +1,522 @@
-.partners-list {
-	width: 100%;
-	margin-bottom: 20px;
-	border: 1px solid #000000;
-	border-collapse: collapse; 
-	background-color: #7de0d3;
+
+body{
+	font-family: "Bakbak One", "Roboto", sans-serif;
```

Comment:
I think you choose wrong default `font-family`. This should be the font that is used for regular text in paragraphs, in this case it is `Poppins` according to design

Reply:
I chose the most frequent font to reduce the amount of code - this approach seemed the most efficient to me
so, the main font should be chosen according to the principle of what font is used in paragraphs?
thanks in advance!

Reply:
Yes, this is a more logical approach in my opinion. Because you should try not to add classes to paragraphs, and their font should be inherited from the parent

Reply:
Thanks! Now its clear) In this case may I just add `Poppins` only for `p`  just so I dont have to redo a lot? :)
thanks in advence!

Reply:
I would still advise using `Poppins` for the entire document) Also you can set `Bakbak One` for h1-h6 tags - this will reduce your workload